pkgs/applications/graphics/photoqt/default.nix

```diff
@@ -30,8 +30,13 @@ stdenv.mkDerivation rec {
     hash = "sha256-kVf9+zI9rtEMmS0N4qrN673T/1fnqfcV3hQPnMXMLas=";
   };
 
+  postPatch = ''
```

Comment:
That's basically https://gitlab.com/lspies/photoqt/-/commit/23e03360adefebadbccfb4ac4f1628caeea216f7. Does the patch apply cleanly here?

Reply:
Oh, didn't see your comment.